### PR TITLE
[WIP] Dracut Multipathd Service Parser

### PR DESCRIPTION
**Pull request by akshay196**
### All Pull Requests:

Check all that apply:

* [X] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [x] Is this PR an enhancement?

### Complete Description of Additions/Changes:

Add parser for the
`/usr/lib/dracut/modules.d/90multipath/multipathd.service` file.

### CODE REVIEW

**Comment by bfahr** [comment]
@akshay196 you need to check the enhancement box if you are adding a new parser.  That would be considered an enhancement to insights-core.

File: insights/parsers/dracut_modules.py [hunk 1]
```diff
@@ -43,3 +43,44 @@ class DracutModuleKdumpCaptureService(IniConfigFile):
         True
     """
     pass
+
+
+@parser(Specs.dracut_multipathd_service)
+class DracutModuleMultipathdService(IniConfigFile):
```
Comment (xiangce) [inline insights/parsers/dracut_modules.py:49]: @akshay196 - Please add this parser to the header of the docstring of this python file.

**Review by xiangce — CHANGES_REQUESTED** [review]

File: insights/parsers/dracut_modules.py (hunk 1 above)
Comment (akshay196) [reply insights/parsers/dracut_modules.py:53]: Added.

**Comment by akshay196** [comment]
Marked as WIP because there is discussion going on related to spec. If we don't want the spec, then I will close this parser PR.

**Pull request closed without merging**